packages/jest/src/utils/config/get-jest-projects.ts

```diff
@@ -0,0 +1,46 @@
+import { readWorkspaceConfig } from '@nrwl/workspace/src/core/file-utils';
+import { dirname, join } from 'path';
+import type { WorkspaceJsonConfiguration } from '@nrwl/devkit';
+
+const JEST_RUNNER_TOKEN = '@nrwl/jest:jest';
+
+function getJestConfigProjectPath(projectJestConfigPath: string): string {
+  return join('<rootDir>', dirname(projectJestConfigPath));
+}
+
+export function getJestProjects() {
+  const ws = readWorkspaceConfig({
+    format: 'nx',
+  }) as WorkspaceJsonConfiguration;
+  const jestConfigurationSet = new Set();
+  for (const projectConfig of Object.values(ws.projects)) {
+    if (!projectConfig.targets) {
+      continue;
+    }
+    for (const targetConfiguration of Object.values(projectConfig.targets)) {
+      if (
+        targetConfiguration.executor === JEST_RUNNER_TOKEN &&
+        targetConfiguration.options?.jestConfig
+      ) {
+        jestConfigurationSet.add(
+          getJestConfigProjectPath(targetConfiguration.options.jestConfig)
+        );
+      }
+      if (targetConfiguration.configurations) {
+        for (const configurationObject of Object.values(
+          targetConfiguration.configurations
+        )) {
+          if (
+            configurationObject.executor === JEST_RUNNER_TOKEN &&
```

Comment:
This doesn't need to be checked here executors aren't assigned to configurations.